src/index.json

```diff
@@ -951,6 +951,52 @@
                     "version": "0.9.0"
                 }
             }
+        ],
+        "loganalytics": [
```

Comment:
Given the command will be `az monitor log-analytics`(with the dash), you may want to rename to `log-analytics` for more clarity.

Reply:
I'm hosting on PyPi and we already have the azure-loganalytics python sdk there, so I was going to follow the azure batch style naming and go with azure-loganalytics-cli-extension. I see that test_index.py validates the package name against the .dist-info metadata.json, so would that (azure-loganalytics-cli-extension) be fine here?

Reply:
Please leave the name as `log-analytics` as this is the name users will see when they run the `az extension list-available` command. This being an extension for azure-cli is implied implicitly.

Batch and others were added before these naming conventions were made. The hosting location does not matter for this.

Reply:
Got it.